app/ethermint.go

```diff
@@ -56,9 +55,9 @@ type (
 		paramsKey   *sdk.KVStoreKey
 		tParamsKey  *sdk.TransientStoreKey
 
-		accountKeeper  auth.AccountKeeper
-		feeCollKeeper  auth.FeeCollectionKeeper
-		coinKeeper     bank.Keeper
+		accountKeeper auth.AccountKeeper
+		feeCollKeeper auth.FeeCollectionKeeper
+		// coinKeeper     bank.Keeper
```

Comment:
Rational?

Reply:
Were existing variables that is unused currently so in case they will be used in the future I have commented out for now. Reason I didn't remove is because it seemed out of the scope of this PR, but I can remove.

Reply:
unsued ATM, the new linter exposed this. Same below